app/style.css

```diff
@@ -15,7 +15,7 @@ body {
 }
 #top {
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-start;
```

Comment:
Yeah just do all the changes and then commit it. Will be included in this pull request